tests/test_vehicle.py

```diff
@@ -81,6 +81,25 @@ def test_permission(self):
 
         self.check(response)
         self.assertEqual(response, data['permissions'])
+        
+    @responses.activate
+    def test_has_permissions(self):
+        data = {
+            "permissions": ["read_odometer" , "read_vehicle_info"]
+        }
+
+        self.queue('GET', 'permissions', data)
+        single_response = self.vehicle.has_permissions("read_odometer")
+        no_required_response = self.vehicle.has_permissions("required:read_odometer")
```

Comment:
super nit: `no_required_response` is a slightly unintuitive name -- maybe even `required_response` would be easier to understand

Reply:
changed in both unit and e2e